docs/included.md

```diff
@@ -13,6 +13,18 @@
   - [API](https://scores.readthedocs.io/en/latest/api.html#scores.continuous.additive_bias)
   - [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Additive_and_multiplicative_bias.html)
   - [Mean Error (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#meanerror)
+* - Consistent Expectile Score
+  - [API](api.md#scores.continuous.consistent_expectile_score)
+  - [Tutorial](project:./tutorials/Consistent_Scores.md)
+  - [Gneiting (2011)](https://doi.org/10.1198/jasa.2011.r101388), [Taggart (2022)](https://doi.org/10.1002/qj.4206)
```

Comment:
```suggestion
  - [Gneiting (2011)](https://doi.org/10.1198/jasa.2011.r101388); [Taggart (2022)](https://doi.org/10.1002/qj.4206)
```
Just for consistency with the rest of the table (which uses semi-colons between multiple references). Sorry I didn't notice this before.

Reply:
Thanks - updated